perfkitbenchmarker/linux_packages/hadoop.py

```diff
@@ -40,9 +40,13 @@
 _URL_OVERRIDE = flags.DEFINE_string(
     'hadoop_bin_url', None, 'Specify to override url from HADOOP_URL_BASE.')
 
+flags.DEFINE_integer('hadoop_blocksize', 256, 'blocksize in mb')
```

Comment:
Expand the description. Get some inspiration from: 
https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/blob/master/perfkitbenchmarker/linux_benchmarks/dpb_terasort_benchmark.py#L85

Save flag definition to a variable and access with it(like we are doing with _URL_OVERRIDE).

Also the default before was 128MB, please leave as it was.

Reply:
Friendly reminder to address this comments.